sky/templates/gcp-ray.yml.j2

```diff
@@ -125,6 +125,8 @@ setup_commands:
     (type -a pip | grep -q pip3) || echo 'alias pip=pip3' >> ~/.bashrc;
   # GCP TPU VM image does not install conda by default.
   - which conda > /dev/null 2>&1 || (wget -nc https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh && bash Miniconda3-latest-Linux-x86_64.sh -b && eval "$(/home/gcpuser/miniconda3/bin/conda shell.bash hook)" && conda init && conda config --set auto_activate_base true);
+  - test -f /home/gcpuser/miniconda3/etc/profile.d/conda.sh && source /home/gcpuser/miniconda3/etc/profile.d/conda.sh || true
+  - conda activate base
```

Comment:
Yes it's required to activate the base env ourselves because we won't re-login and source the bashrc when doing  `do_filemounts_and_setup_on_local_workers` https://github.com/skypilot-org/skypilot/pull/1001/files#diff-ca7f4bd7a7474f38571ee8faf61430a6cc38c616bcdbec6e1dda227f9b98c3f4R179